test/e2e/quadlet_test.go

```diff
@@ -788,229 +793,229 @@ BOGUS=foo
 
 	DescribeTable("Running quadlet test case",
 		runQuadletTestCase,
-		Entry("Basic container", "basic.container", 0, ""),
```

Comment:
I put it in a separate commit in order to make it more readable. I guess it didn't work :).
Originally, I thought I'd add this feature to other unit types, but, I'm not sure it's really needed for the units that are intended to be used as dependencies.
I can look into separating this test if you think it's better

Reply:
sure a separate commit helps but the point you still touch a lot of lines for no good reason, duplicating  `...0, "", ""),` is annoying to see and makes it harder to read.

So thinking about this there should really be a way to pass optional arguments. Because the next time someone adds a new arg I don't want to see such a bug diff again.